src/SimpleGraphs/generators/staticgraphs.jl

```diff
@@ -763,3 +763,25 @@ function lollipop_graph(n1::T, n2::T) where {T <: Integer}
     add_edge!(g, n1, n1+1)
     return g
 end
+
+"""
+    friendship_graph(n)
+Create a [friendship graph](https://en.wikipedia.org/wiki/Friendship_graph) 
+consisting of n copies of the cycle graph C3 with a common vertex`.
+"""
+
+function friendship_graph(n::T) where {T <: Integer} 
+    n <= 0 && return SimpleGraph(1)
+    g = SimpleGraph(2 * n + 1)
```

Comment:
```suggestion
    n <= 0 && return SimpleGraph{T}(1)
    g = SimpleGraph{T}(2 * n + 1)
```
Otherwise this function is not type stable. If, for example, `n` is of type `UInt`, then `friendship_graph(UInt(0))` would return a graph of type `SimpleGraph{Int}` but `friendship_graph(UInt(1))` would return a graph of type `SimpleGraph{UInt}`.